fixtures/simple-vite/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@vocab-fixtures/simple-vite",
+  "version": "1.0.0",
+  "author": "SEEK",
+  "private": true,
+  "scripts": {
+    "compile": "vocab compile",
+    "start": "vocab compile --watch & vite",
+    "build": "vocab compile && vite build"
+  },
+  "dependencies": {
+    "@babel/core": "^7.12.0",
+    "@types/react": "^18.0.9",
+    "@types/react-dom": "^18.0.4",
+    "@vocab/cli": "workspace:*",
+    "@vocab/core": "workspace:*",
+    "@vocab/pseudo-localize": "workspace:*",
+    "@vocab/react": "workspace:*",
+    "@vocab/webpack": "workspace:*",
+    "babel-loader": "^9.1.2",
```

Comment:
This dep isn't needed for this fixture.
```suggestion
```

Reply:
Good catch. Thought I removed all unnecessary deps.

Reply:
I'm removing the `@vocab/webpack` dep also. Not needed in `vite`